apps/web/src/pages/ProfileAndApplicationsPage/components/TrackApplications/TrackApplicationsCard.tsx

```diff
@@ -1,122 +1,218 @@
 import * as React from "react";
 import { notEmpty } from "@gc-digital-talent/helpers";
-import { getLocale } from "@gc-digital-talent/i18n";
-import { Chip, Chips, Heading, HeadingProps } from "@gc-digital-talent/ui";
+import { Heading, HeadingProps, Pill, Separator } from "@gc-digital-talent/ui";
 import { useIntl } from "react-intl";
-import { PoolCandidate, Skill } from "~/api/generated";
-import ApplicationActions from "~/pages/Applications/MyApplicationsPage/components/ApplicationCard/ApplicationActions";
+import { PoolCandidate } from "~/api/generated";
+import ApplicationActions, {
+  DeleteActionProps,
+} from "~/pages/Applications/MyApplicationsPage/components/ApplicationCard/ApplicationActions";
 import {
   isDraft,
   isExpired,
-  isPlaced,
 } from "~/pages/Applications/MyApplicationsPage/components/ApplicationCard/utils";
 import { getFullPoolTitleHtml } from "~/utils/poolUtils";
-import TrackApplicationsStatus from "./TrackApplicationsStatus";
+import { getStatusPillInfo } from "~/components/QualifiedRecruitmentCard/utils";
+import ApplicationLink from "~/pages/Pools/PoolAdvertisementPage/components/ApplicationLink";
+import useMutations from "~/pages/Applications/MyApplicationsPage/components/ApplicationCard/useMutations";
+import { getRecruitmentType, isQualifiedStatus } from "~/utils/poolCandidate";
+import ShieldCheckIcon from "@heroicons/react/20/solid/ShieldCheckIcon";
+import { PoolCandidateStatus } from "@gc-digital-talent/graphql";
+import { useAuthorization } from "@gc-digital-talent/auth";
+import { getApplicationDateInfo } from "./utils";
 
 export type Application = Omit<PoolCandidate, "user">;
 
 export interface TrackApplicationsCardProps {
   application: Application;
   headingLevel?: HeadingProps["level"];
+  onDelete: DeleteActionProps["onDelete"];
 }
 
 const TrackApplicationsCard = ({
   application,
   headingLevel = "h2",
+  onDelete,
 }: TrackApplicationsCardProps) => {
   const intl = useIntl();
-  const locale = getLocale(intl);
-
-  // Essential Skills
-  const skills = application.pool.essentialSkills
-    ? application.pool.essentialSkills
-    : [];
 
   // Conditionals for card actions
   const applicationIsDraft = isDraft(application.status);
   const recruitmentIsExpired = isExpired(
     application.status,
     application.pool.closingDate,
   );
-  const isApplicantPlaced = isPlaced(application.status);
+  const isDraftExpired = applicationIsDraft && recruitmentIsExpired;
+  const isApplicantQualified = isQualifiedStatus(application.status);
+
+  // We don't get DraftExpired status from the API, so we need to check if the draft is expired ourselves
+  const statusPill = isDraftExpired
+    ? getStatusPillInfo(PoolCandidateStatus.DraftExpired, intl)
+    : getStatusPillInfo(application.status, intl);
+
+  const applicationDateInfo = getApplicationDateInfo(application, intl);
+  const { user } = useAuthorization();
+  const applicationTitle = getFullPoolTitleHtml(intl, application.pool);
```

Comment:
Shouldn't this be called `poolTitle` instead of `applicationTitle` since it is calling the `getFullPoolTitleHtml` function?

Reply:
hmm its named based on its usage-  title for the application card which came from `getFullPoolTitleHtml`